en/docs/install-and-setup/setup/deployment-best-practices/performance-tests-results-graphql.md

```diff
@@ -0,0 +1,162 @@
+# Performance Test Results
+
+The performance of WSO2 API Manager was measured for GraphQL APIs. While the schema, the queries, and the responses were mimicked using the [Starwars API](https://github.com/wso2/samples-apim/tree/master/graphql-backend), the backend used for the test is a Netty backend. The netty service is configured so that the response is aligned with the GraphQL request in order to only provide the data requested by a particular query.
```

Comment:
```suggestion
The performance of WSO2 API Manager was measured for GraphQL APIs. While the schema, the queries, and the responses were mimicked using the [Starwars API](https://github.com/wso2/samples-apim/tree/master/graphql-backend), the backend used for the test is a Netty backend. The Netty service is configured so that the response is aligned with the GraphQL request in order to only provide the data requested by a particular query.
```